integration/Jenkinsfiles/ftest.groovy

```diff
@@ -88,12 +83,16 @@ def emitVerifyClosure(String nodelabel, String sha, String zipfile, String name,
         node(nodelabel) {
             stage(name) {
                 ws("${WORKSPACE}-${name}") {
+                    def jdk = tool name: 'java-11-openjdk'
+
+                    def timeoutHours = params.NX_TIMEOUT_HOURS == null ? '3' : params.NX_TIMEOUT_HOURS
```

Comment:
In groovy this can be simplified to ` def timeoutHours = params.NX_TIMEOUT_HOURS ?: '3'`